posix.mak

```diff
@@ -28,6 +30,12 @@
 
 QUIET:=
 
+ifeq ($(DEBUGGER),)
+DEBUGGERLOCAL=gdb
+else
+DEBUGGERLOCAL=$(DEBUGGER)
+endif
```

Comment:
`DEBUGGERLOCAL` seems a bit unnecessary - why is it needed?
`DEBUGGER` can be overwritten and will be passed by default to the invoked Makefile instance ;-)